_config.yml

```diff
@@ -192,3 +192,9 @@ share:
   pt: Compartilhe essa página
   bn: পৃষ্ঠা শেয়ার করুন
   vi: Chia sẻ trang này
+
+warnings:
+  en:
+    major: "Major differences"
```

Comment:
Name them what you want so that everyone can see them from the start. A sensible default is usually a good thing (Rails has corrupted me). If there is a better way to express it, someone will think about it.